doc/src/vpr/debug_aids.rst

```diff
@@ -103,3 +103,42 @@ Available Operators
           * Ex. in_blocks_affected == 11 || temp_count == 9
   * +=
           * Ex. move_num += 8
+          
+          
+Setting moves maually during placement
+======================================
+
+Overview
+~~~~~~~~
+
+The manual move generator, when enabled, allows the user to specify the next move in placement. The user can set a breakpoint to stop the placer at a specific point and start making manual moves. After specifying the move, the user is presented with the costs of the move and can then decide to accept or reject it.
+
+How to use the manual move generator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, you should make sure the manual move option is enabled, as seen in the figure.
+
+.. image:: https://www.verilogtorouting.org/img/manualMoveCheckbox.png
+    :align: center
+    
+Then after pressing the “Proceed” button a window pops up that asks for the block you want to move, and the location you would want to move it to. 
+
```

Comment:
location you *want to move it to.* (delete would)

Reply:
resolved